.github/workflows/branch_deployments.yml

```diff
@@ -9,6 +15,7 @@ concurrency:
 
 env:
   DAGSTER_CLOUD_URL: "http://dagster.cloud/metaplane"
```

Comment:
should be `https://metaplane.dagster.cloud`

Reply:
ok should it also be this way for the deploy.yml file? I assume so

Reply:
yes in both places